api/app/controllers/organization.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+from uuid import UUID
+from sqlalchemy.orm import Session
+from app.models.organization import Organization
+from app.schemas.organization_schemas import OrganizationCreate, OrganizationRead, OrganizationReadWithUsers
```

Comment:
since the OSS doesn't actually support Organizations (yes technically there is 1 org but it gets auto-applied to every user) let's move this Controller and the matching route to the .com repo